src/fabric_cli/core/fab_constant.py

```diff
@@ -316,3 +316,16 @@
     "workspaceId",
     "folderId",
 }
+
+# Item set constants
+ITEM_QUERY_DEFINITION = "definition"
+ITEM_QUERY_PROPERTIES = "properties"
+ITEM_QUERY_DISPLAY_NAME = "displayName"
+ITEM_QUERY_DESCRIPTION = "description"
+
+# Allowed metadata keys for item set operations
+ITEM_SET_ALLOWED_METADATA_KEYS = [
```

Comment:
what is the reason of maintaing this? 

what about the other types like workspace / capacity / sparkpool. do we need whitelist there as well?

Reply:
We will probably use something similar per wokrspace / capacity / spark pool. For example - we support setting sku.name for capacities.

I can either hard-code it directly in the setItem method or define it in the constants file. I don’t see any strong advantages or disadvantages to placing it in one location versus the other.